common/timedmutex/timed_mutex_test.go

```diff
@@ -5,10 +5,41 @@ import (
 	"time"
 )
 
+// 1000000	        1074 ns/op	     136 B/op	       4 allocs/op (prev)
+// 2423571	       503.9 ns/op	       0 B/op	       0 allocs/op (current)
 func BenchmarkTimedMutexTime(b *testing.B) {
-	tm := NewTimedMutex(20 * time.Millisecond)
+	tm := NewTimedMutex(0)
+	for i := 0; i < b.N; i++ {
+		tm.LockForDuration()
+	}
+}
+
+// 352309195	         3.194 ns/op	       0 B/op	       0 allocs/op (prev)
+// 927051118	         1.298 ns/op	       0 B/op	       0 allocs/op
+func BenchmarkTimedMutexTimeUnlockNotPrimed(b *testing.B) {
+	tm := NewTimedMutex(0)
+	for i := 0; i < b.N; i++ {
+		tm.UnlockIfLocked()
+	}
+}
+
+// 95322825				15.51 ns/op	       0 B/op	       0 allocs/op (prev)
+// 239158972			4.621 ns/op	       0 B/op	       0 allocs/op
```

Comment:
No idea what is going on here. 
<img width="460" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/045a9c45-6e4e-4e5d-9153-d239c91f4be4">

Reply:
In what sense?

Reply:
<img width="598" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/44330e0d-63eb-4752-87f3-72993635aaef">
Maybe its just a github me issue the formatting of these comments

Reply:
oh, in how tabs are rendered 😄 wasn't sure you if you were commenting on the benchmark design itself